### DISCUSSION TITLE
how to use RayTracingSceneSrg and RayTracingMaterialSrg in RayTracing pipeline?

### DISCUSSION THREAD

**Discussion by ludi1991** [o3de/o3de#11302] [Help Wanted]
AtomsampleViewer have some samples (RTAO) but crash happens when I tried to add RayTracingSceneSrg and RayTracingMaterialSrg in RTAO.Only RTXGI uses RayTracingSceneSrg and RayTracingMaterialSrg  but shaders are compiled so there is no shader source code. Is there any sample about how to use RayTracingSceneSrg and RayTracingMaterialSrg in rt shader? RayTracingPass has no sample code too..

**Comment by siliconvoodoo** [comment]
How about o3de\Gems\Atom\Feature\Common\Code\Source\DiffuseGlobalIllumination\DiffuseProbeGridRayTracingPass.cpp ?
